src/__tests__/libhoney_test.ts

```diff
@@ -1,25 +1,11 @@
 /* eslint-env node, jest */
 import libhoney from "../libhoney";
+import superagent from "superagent";
+import superagentMocker from "superagent-mocker";
 
-let superagent = require("superagent");
-let mock = require("superagent-mocker")(superagent);
+let mock = superagentMocker(superagent);
 
 describe("libhoney", () => {
-  describe("constructor options", () => {
```

Comment:
This test needs rewritten. At first pass I was still trying to stabalise the types and this test was causing bother.